### DISCUSSION TITLE
一个玩家下线了, 他的Avatar实体的id，以后会给其他玩家使用不？

### DISCUSSION THREAD

**Discussion by ybyaoyao** [kbengine/kbengine#1448] [General]
一个玩家下线了, 他的Avatar实体的id，以后会给其他玩家使用不？
比如玩家avatar的id是 9，然后他下线了，以后这个9会不会给新的实体id使用呢？

谢谢。因为我记录了某些玩家的id, 如果不会给后来的玩家使用，我就不用去删除这些下线了的玩家的id了。

**Comment by kaisong524118** [comment]
用databasesId

**Reply by huhubis** [reply]
实体id不可重用
